perceval/backends/core/github.py

```diff
@@ -85,8 +85,9 @@ class GitHub(Backend):
         from the GitHub public site.
     :param tag: label used to mark the data
     :param archive: archive to store/retrieve items
+    :param exclude_user_data: Don't fetch personal user information from the API
```

Comment:
Please start the sentence with lower case and do not use contractions. (`do not fetch`